packages/apps/job-launcher/server/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
-    "target": "ES2020",
+    "target": "ES2022",
```

Comment:
By some reason using `ErrorOptions` (which is part of es2022) worked for JL when it shouldn't. I spend many hours trying to understand why it happened and had no luck, probably something just messed up in global monorepo setup.

Set proper target to avoid future confusion